stable/minio/templates/ingress.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     chart: {{ template "minio.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- with .Values.ingress.customlabels }}
```

Comment:
shouldn't this be `{{- if .Values.ingress.customlabels }}` ?

Reply:
these both are valid, I can opt for `if` syntax if more relevant

```
{{- with .Values.ingress.customlabels }}
{{ toYaml . | indent 4 }}
{{- end }}
```

```
{{- if .Values.ingress.customlabels }}
{{ toYaml .Values.ingress.customlabels | indent 4 }}
{{- end }}
```